router/router_test.go

```diff
@@ -313,3 +391,62 @@ func wsServer(t *testing.T) {
 	}
 	log.Println("Out of server")
 }
+
+func TestUnitGetRouterFromConfigService(t *testing.T) {
+	t.Log("Testing: Retrieving router port from the config service, after router service startup")
+	expectedValue := "5002"
+	client := &http.Client{}
+	request, _ := http.NewRequest("GET", ConfigServiceUrl +"/config-service/variable/router_http_port", nil)
+	response, err := client.Do(request)
```

Comment:
Use http.DefaultClient.Get , it is a single line statement replacing all three of the above lines